bin/maintenance/build-wheel.py

```diff
@@ -235,22 +235,35 @@ def cli(obj, target_dir):
 @click.option('--no-assets', 'assets', is_flag=True, flag_value=False, default=True, help='skip building assets')
 @click.option('--add-version-suffix', is_flag=True, help='Add a local suffix (+yyyymmdd.hhmm.commit) to the version')
 @click.option('--ignore-unclean', is_flag=True, help='Ignore unclean working tree')
+@click.option('--no-git', is_flag=True, help='Skip checks and assume clean non-git working tree')
```

Comment:
```suggestion
@click.option('--no-git', is_flag=True, help='Skip git checks and assume clean working tree')
```

same as my previous suggestion, i just thought you'd update it in both places ;)